bundler/spec/dependabot/bundler/file_parser_spec.rb

```diff
@@ -507,44 +446,20 @@
     end
 
     context "with a Gemfile that includes a file with require_relative" do
-      let(:files) { [gemfile, lockfile, required_file] }
-      let(:gemfile_fixture_name) { "includes_require_relative" }
-      let(:lockfile_fixture_name) { "Gemfile.lock" }
-      let(:required_file) do
-        Dependabot::DependencyFile.new(
-          name: "../some_other_file.rb",
```

Comment:
Looks like this was dropped from the fixture folder but not sure how to do this without creating the dependency file like this. Assuming this is returned from the file fetcher if setting up updates in a nested directory with a Gemfile that requires a file from the parent directory.

Reply:
Yeah this now lives in the `bundler1` folder which isn't great but works

Reply:
Could you nest the project within the project and move the required file down?

Reply:
e.g. `project_dependency_files("bundler1/includes_require_relative_gemfile/nested")`

Reply:
Yeah that's what I was trying to get at, I think that'll work 👍